kubernetes/linux/setup.sh

```diff
@@ -39,7 +39,7 @@ mv /usr/lib/ruby/gems/3.1.0/specifications/default/uri-0.11.0.gemspec /usr/lib/r
 gem uninstall time --version 0.2.0
 gem uninstall uri --version 0.11.0
 
-sudo tdnf install -y azure-mdsd-1.27.4
+sudo tdnf install -y azure-mdsd-1.28.11
```

Comment:
We should also update agentVersion with MDSD version in the yaml. This annotation for the customer to check if AMA agent version. This was introduced when the critical security vulns found in base omsagent  and customer couldnt figure out which version of the agent has vulns.

Reply:
done